spec/request/posts_controller_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+

```

Comment:
- [ ] Great job creating `users_spec.rb` and `posts_spec.rb` in `spec/request` directory, however, let's pluralize the file directory to `requests` since this will contain more than a single file.